README.md

```diff
@@ -122,10 +188,15 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Your Answer:**
 
+All Users, Create a New Post, and Logout links appear in upper right.
+
+
 * **Question:** What happens if you enter in the incorrect information? What _should_ happen?
 
 * **Your Answer:**
 
+The token is "undefined".
```

Comment:
We should receive some kind of feedback message and be redirected to the login page.